doc/classes/SkeletonProfileHumanoid.xml

```diff
@@ -5,6 +5,63 @@
 	</brief_description>
 	<description>
 		A [SkeletonProfile] as a preset that is optimized for the human form. This exists for standardization, so all parameters are read-only.
+		A humanoid skeleton profile is structured as follows:
+		[codeblock]
+		Root
+		└─ Hips
+		    ├─ LeftUpperLeg
+		    │  └─ LeftLowerLeg
+		    │     └─ LeftFoot
+		    │        └─ LeftToes
+		    ├─ RightUpperLeg
+		    │  └─ RightLowerLeg
+		    │     └─ RightFoot
+		    │        └─ RightToes
+		    └─ Spine
+		        └─ Chest
+		            ├─ UpperChest
+		            │  ├─ LeftShoulder
+		            │  │  └─ LeftUpperArm
+		            │  │     └─ LeftLowerArm
+		            │  │        └─ LeftHand
+		            │  │           ├─ LeftThumbMetacarpal
+		            │  │           │  └─ LeftThumbProximal
+		            │  │           ├─ LeftIndexProximal
+		            │  │           │  └─ LeftIndexIntermediate
+		            │  │           │    └─ LeftIndexDistal
+		            │  │           ├─ LeftMiddleProximal
+		            │  │           │  └─ LeftMiddleIntermediate
+		            │  │           │    └─ LeftMiddleDistal
+		            │  │           ├─ LeftRingProximal
+		            │  │           │  └─ LeftRingIntermediate
+		            │  │           │    └─ LeftRingDistal
+		            │  │           └─ LeftLittleProximal
+		            │  │              └─ LeftLittleIntermediate
+		            │  │                └─ LeftLittleDistal
+		            │  └─ RightShoulder
+		            │     └─ RightUpperArm
+		            │        └─ RightLowerArm
+		            │           └─ RightHand
+		            │              ├─ RightThumbMetacarpal
+		            │              │  └─ RightThumbProximal
+		            │              ├─ RightIndexProximal
+		            │              │  └─ RightIndexIntermediate
+		            │              │     └─ RightIndexDistal
+		            │              ├─ RightMiddleProximal
+		            │              │  └─ RightMiddleIntermediate
+		            │              │     └─ RightMiddleDistal
+		            │              ├─ RightRingProximal
+		            │              │  └─ RightRingIntermediate
+		            │              │     └─ RightRingDistal
+		            │              └─ RightLittleProximal
+		            │                 └─ RightLittleIntermediate
+		            │                   └─ RightLittleDistal
+		            └─ Neck
```

Comment:
Neck is a child of UpperChest and should be listed before LeftShoulder / RightShoulder for readability.